### ISSUE TITLE
Keyboard doesn't pop up automatically when receiving to LN

### ISSUE THREAD

**Issue by rkfg** [btcontract/wallet#64]
Steps to reproduce:

1. Press Receive
2. Receive to Lightning invoice
3. A form appears and the cursor is blinking in the SAT field but the on-screen keyboard doesn't appear
4. I have to tap this field so the numeric keyboard pops up, this should happen automatically imo

**Comment by btcontract** [comment]
It's possible to make it pop up automatically, the only thing which stops me is that there are two editable fields: one for bitcoins and another for fiat. Generally software should proceed with such things only if there is one possible action to take, but user may choose either of those fields.

I don't have strong feelings about it though, if it still feels really wrong like it is now then please confirm and I'll make a keyboard to pop up.

**Comment by btcontract** [comment]
Fixed in https://github.com/btcontract/wallet/commit/3e1b77679ce7fbdb2f8b421cb203150dc7fbc754.

**Comment by rkfg** [comment]
I see that in the current release this is fixed. Can you add the same logic to LNURL Pay so that the keyboard pops up automatically because the amount field is already focused?

**Comment by btcontract** [comment]
Fixed again in https://github.com/btcontract/wallet/commit/b9c1703a056268fe0ec76bb7b7d123cca87e3fb4. After some testing I did not like how automatic keyboard was working as it was hiding important popup details on smaller screens which users better notice at start.

So the latest solution to this is to not focus on input fields instead whenever user has more than one action option (such as start entering an amount, or tap MAX, or provide a comment etc). In all other popups with single input field keyboard still pops up automatically.
